src/seedu/addressbook/Main.java

```diff
@@ -103,7 +106,9 @@ private void recordResult(CommandResult result) {
      * @param command user command
      * @return result of the command
      */
-    private CommandResult executeCommand(Command command)  {
+    private CommandResult executeCommand(Command command) throws NoStorageFileException  {
+    	if (storage.getPath().equals(null))
```

Comment:
You are using the exception in a normal flow. 
If you check the something regularly, you cannot call it an exception.
And particularly, the exception should be threw from the method you called.